public/blog/jsbin-toxic-part-4.md

```diff
@@ -116,4 +116,4 @@ The next time around, with products like [Confwall](https://confwall.com), I lau
 
 ---
 
-Part 5 sees our conclusion and tells of the police encounters I've had, along with the worst email I've ever received.
\ No newline at end of file
+Part 5 sees our conclusion and tells of the police encounters I've had, along with the worst email I've ever received.
```

Comment:
Sorry - github picked this line up as altered, but I've not touched it. I have to assume it's a whitespace change. I should have cleaned it up before creating the PR. Soz.